ivy/data_classes/array/activations.py

```diff
@@ -40,6 +40,7 @@ def relu(
 
         Examples
         --------
+
```

Comment:
Why are we adding a newline here? Why only after examples, but not returns, parameters? You failing the format checks :sweat_smile:  D412: No blank lines allowed between a section header and its content ('Examples')